<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickTest</class>
 <widget class="QDialog" name="QuickTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_Free_quiz_test">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1001</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:   rgb(85, 85, 127)</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>401</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  	#F8F8FF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>19</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>1. How soon after you wake up do you have</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>your first cigarette?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Button_1_S_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>A. Within 5 minutes </string>
      </property>
     </widget>
     <widget class="QPushButton" name="B_Button_1_S_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>B. 6-30 minutes </string>
      </property>
     </widget>
     <widget class="QPushButton" name="C_Button_1_S_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>C. 31-60 minutes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="D_Button_1_S_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>D. After 60 minutes </string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>700</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>401</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  	#F8F8FF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>2. Which cigarette would you hate </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>most  to give up?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Button_2_S_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>A. The first one in the morning </string>
      </property>
     </widget>
     <widget class="QPushButton" name="B_Button_2_S_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>B. All others </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QFrame" name="frame_11">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>411</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  	#F8F8FF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>3. How many cigarettes do you smoke </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>per day?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Button_3_S_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>A. 10 or fewer </string>
      </property>
     </widget>
     <widget class="QPushButton" name="B_Button_3_S_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>B. 11-20 </string>
      </property>
     </widget>
     <widget class="QPushButton" name="C_Button_3_S_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>C. 21-30 </string>
      </property>
     </widget>
     <widget class="QPushButton" name="D_Button_3_S_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>D. 31 or more </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QFrame" name="frame_12">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>401</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  	#F8F8FF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>4. Do you smoke more often during</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>the first hours after waking</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Button_4_S_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>A. Yes </string>
      </property>
     </widget>
     <widget class="QPushButton" name="B_Button_4_S_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>B. No </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>than during the rest of the day?</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QFrame" name="frame_13">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>401</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  	#F8F8FF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>5. Do you smoke even if you are so ill</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>that you are in bed most of the day?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Button_5_S_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>A. Yes </string>
      </property>
     </widget>
     <widget class="QPushButton" name="B_Button_5_S_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>B. No </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame_15">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>401</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  	#F8F8FF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>6. Do you find it difficult to refrain from</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>smoking in places where it is</string>
      </property>
     </widget>
     <widget class="QPushButton" name="A_Button_6_S_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>A. Yes </string>
      </property>
     </widget>
     <widget class="QPushButton" name="B_Button_6_S_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}
</string>
      </property>
      <property name="text">
       <string>B. No </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>forbidden eg, in Mosque compound,</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>the library, the Home, etc?</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QFrame" name="frame_14">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>401</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:  	#F8F8FF</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="l_Situation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:  rgb(170, 0, 0);
font: 700 15pt &quot;Segoe UI&quot;;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Home_Button_from_Quiz">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton
{
font: 14pt &quot;Segoe UI&quot;;
}
QPushButton:hover
{
color:white;
border-radius:20px;
background:rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_Show_Score">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>91</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(80, 159, 239);
font: 72pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Score is</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Your Score Means ....</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_Show_Score_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>31</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(80, 159, 239);
font: 72pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>401</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Segoe UI&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>    Nicotine Dependence Test</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
